src/main/java/net/sf/jabref/shared/security/Password.java

```diff
@@ -0,0 +1,68 @@
+package net.sf.jabref.shared.security;
+
+import java.security.GeneralSecurityException;
+import java.util.Base64;
+
+import javax.crypto.Cipher;
+import javax.crypto.spec.SecretKeySpec;
+
+/**
+ *   {@link Password} contains methods which are useful to encrypt and decrypt passwords using symetric algorithms.
+ */
+public class Password {
+
+    private static final String ALGORITHM = "AES";
```

Comment:
Please specify transformation/padding/mode : 

> It is recommended to use transformations that fully specify the algorithm, mode, and padding instead of relying on the defaults.

As noted here: https://docs.oracle.com/javase/8/docs/technotes/guides/security/SunProviders.html